platform/import-export/src/actions/ImportAsJSONAction.js

```diff
@@ -154,7 +154,8 @@ define(['zepto', 'objectUtils'], function ($, objectUtils) {
         tree = JSON.stringify(tree).replace(new RegExp(oldIdKeyString, 'g'), newIdKeyString);
 
         return JSON.parse(tree, (key, value) => {
-            if (Object.prototype.hasOwnProperty.call(value, 'key')
+            if (value !== null
```

Comment:
how about `undefined`?

Reply:
Do you mean I should add value !== undefined as well?

Reply:
Done.